packages/theme/src/themes/fram-theme/theme.ts

```diff
@@ -16,6 +16,20 @@ const contrastColor = (
   };
 };
 
+// Currently exactly the same as in the AtB theme.
+// The colors can be changed, but should follow standard practice:
+// no entry - dark/black
+// no parking - red
+// slow zone - yellow
+// allowed - blue
+
+export const geofencingZoneBaseColors = {
+  blue_500: contrastColor('#007C92', 'light'),
+  yellow_100: contrastColor('#F0E973', 'dark'),
+  red_400: contrastColor('#C76B89', 'dark'),
+  red_900: contrastColor('#380616', 'light'),
+};
```

Comment:
@mortennordseth This (partially) introduces the base colors concept to non-AtB themes. Is that fine, or do you prefer that we add `contrastColor()` directly in `themes`?

Reply:
We're actually in the process of implementing base colors for FRAM and NFK so this is fine :)